app/assets/javascripts/components/sidebar_item.es6.jsx

```diff
@@ -1,28 +1,38 @@
 class SidebarItem extends React.Component {
 
-	get styles() {
+  static get propTypes() {
+    return {
+      label: React.PropTypes.string.isRequired,
+      icon: React.PropTypes.string.isRequired,
+    };
+  }
```

Comment:
When we define `propTypes` we should also define `defaultProps`!
Check the `GeneralButton` component for an example.